src/lib/EventPublisher.js

```diff
@@ -77,9 +77,9 @@ class EventPublisher extends EventEmitter {
 
 
   subscribe = (
-    eventName: ?string = ALL_EVENTS,
+    eventName: string = ALL_EVENTS,
```

Comment:
tried to change the order, doesn't have any outcome. eventName: ?string = ALL_EVENTS - this doesn't work.

this works though, but seems doesn't right:
eventName?: string = ALL_EVENTS,
filterOptions?: FilterOptions = {},

because i want to use it like so

      this._eventPublisher.subscribe(
        device.onCoreEvent,
        messageName,
        /* filterOptions */null,
        deviceID,
      );

but get an error :/